lib/trackler/problem.rb

```diff
@@ -49,6 +49,31 @@ def yaml_url
       end
     end
 
+    # Used by exercism.io/app/views/languages/_contribute_exercise.erb
+    def readme_url
+      md_url
+    end
+
+    # Used by exercism.io/app/views/languages/_contribute_exercise.erb
+    def data
+      test_data_url
+    end
+
+    def test_data_url
+      json_url
+    end
+
+    def implementations
```

Comment:
Exercism should access `Trackler.implementations` directly. I'll make the change there.